constitution.md

```diff
@@ -20,126 +21,96 @@ Team.
+   preference or disability.
+   
+### Dues
+
+The SDSM&T Aerial Vehicle Team believes that the work put in by the
+students is sufficient and will not impose any additional dues or fees
+upon its members.  All funding will be sought through other means such
+as research grants, school funding, CAMP funding, Student Association
+funding, corporate sponsorship, etc.
+
+### Qualifications
 
-1. Chief Engineer - Presides over general meetings when the team
-   manager is absent.  Approves projects for members to be involved
-   in.  Will have final approval of design decisions.  Will oversee
-   the overall construction of all projects.  Acts as a liaison for
-   companies that wish to be involved in supporting the team.  Will
-   approve all orders that involve outgoing money exceeding $50.  CAMP
-   advisors must approve the selection of the chief engineer.
-
-2. Team Manager - Organizes team leaders of all projects to make sure
-   the projects are being executed as planned.  Presides over all
-   general meetings.  Manages the team’s time-line and maintains team
-   organization.
-
-3. Chief Information Officer - Will manage all of the information and
-   documents produced and required by the UAV team.  Will be
-   responsible for keeping track of membership. Will manage the files
-   and file structure on the school servers, including project
-   management systems and mailing lists.
-
-4. Recording Secretary - Will take minutes during meetings as well as
-   taking attendance.  Will aid other officers in fulfillment of their
-   duties.
-
-5. Webmaster - Will manage the team website.
-
-6. Treasurer - Will account for the flux of all funds coming and
-   leaving the team’s money accounts.  Will ensure that the team stays
-   within its budget.  Will be responsible for completing financial
-   reports that detail the team’s financial position at the end of
-   each semester and when demanded from the team’s supporters.  Along
-   with the Chief Engineer and Advisor, will approve all orders for
-   materials totaling more than $50.
-
-7. Public Relations Chair - Will organize promotions of the SDSM&T
-   Unmanned Aerial Vehicle Team in the SDSM&T community.  Will act as
-   the team’s representative to the Student Association. Will organize
-   programs that involve helping the surrounding community.  Will be
-   responsible for creating guidelines for groups attending
-   competitions to best promote SDSM&T in the robotics community.
-   Will be responsible for all press releases.
-
-## ARTICLE V. ELECTION OF OFFICERS
-
-1. Officers will be selected through a process of being nominated by
+There are no basic qualifications for entry into the team other than
+a time commitment and intangibles such as drive, ingenuity, and a desire
+to excel.  However, the UAV team's mission and competition level is such
+that students below a Junior standing will have trouble contributing

```

Comment:
I don't know if stating that some students may be limited, and that Junior level education is necessary is wise.  I don't see a time in which someone will go back to the this line in the Constitution for reference on any matter.  Perhaps we just remove this line.